jeremydaly/dynamodb-toolbox
# ci/test new workflow

- install prettier
- install and run prettier on all files
- implement type infering in Entities
- update tests following type infering
- add type-infering unit test
- types: support GSIs
- fix: entity update tests
- improve GSI support
- support execute & parse options in inferred types
- Support timestamp option in inferred types
- export GetOptions, PutOptions, DeleteOptions, QueryOptions & UpdateOptions utility types
- fix bug in querying
- fix entity.update.unit.test
- update package-lock.json
- fix type assumption causing infinite loop in type check
- fix type-check errors & warnings
- support hidden option in inferred entity method types
- reintroduce projectionExpressions in Table query only
- improve scan and query type infering support
- update README to add type infering documentation
- fix transform type
- export EntityItem utility type
- version bump
- update packages
- version bump
- update readme
- fix query table in readme
- add support for dependsOn keyword in Entity type infering
- fix bug in querying on falsy values
- fix: ParsedAttributes ts error
- bump alpha version
- fix stability issue and README
- readme update
- fix jest config and update packages
- bump version
- initial commit
- added more node versions
- changed workflow file name
- added type check
